package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Mon Jan 15 14:22:32 UTC 2018 - jlopez@suse.com
+
+- Added sanity checks for partitioning setup.
+- Partitioner: setup issues are shown to the user before continue.
+  Mandatory product volumes are required according to control file.
```

Comment:
I miss the storage-ng fate entry and also mentioning the several bug reports we have about the missing checks.